extensions/Mist/KeyHistory.js

```diff
@@ -0,0 +1,169 @@
+// Name: Key History
+// ID: KeyHistoryExtension
```

Comment:
please include your name somewhere in the extension ID to prevent overlap, something like "MistKeyHistory" would work.

Reply:
yea, a lot of this is kinda old, i do normally include my name in the id on all my newer extensions

Reply:
done